src/base-command.js

```diff
@@ -18,9 +18,13 @@ const yaml = require('js-yaml');
 const path = require('path');
 const libRuntime = require("@adobe/aio-lib-runtime");
 const debug = require('debug')('aio-asset-compute.base');
+const {loadConfig} = require('@adobe/aio-cli-plugin-app/src/lib/config-loader');
 // imported like this so that we can overwrite child_process.spawnSync in unit tests
 const child_process = require("child_process");
 
+const ASSET_COMPUTE_EXT_PATH = "dx/asset-compute/worker/1"; // path to asset compute extension
```

Comment:
Where does the value `dx/asset-compute/worker/1` come from?

Reply:
(This looks like a debugging value... ? Shouldn't it be removed?)

Reply:
it is the defined path to find our extension in the config loader by aio team. it was suggested by the comment here: https://github.com/adobe/generator-aio-app/issues/157#issuecomment-891682315

Reply:
(see here: https://www.adobe.io/project-firefly/docs/guides/extensions/#aem-asset-microservices-extension-point)
I also asked for confirmation this path won't change.
it seems like if we update the version, it will change to `"dx/asset-compute/worker/2"`